tls/s2n_client_cert.c

```diff
@@ -101,6 +101,7 @@ static S2N_RESULT s2n_client_cert_chain_store(struct s2n_connection *conn,
 
 int s2n_client_cert_recv(struct s2n_connection *conn)
 {
+    uint32_t cert_start = conn->handshake.io.read_cursor;
```

Comment:
nit - cert_start might be a bit misleading since there a length values as well. Maybe `cert_message_start`?
```suggestion
    uint32_t cert_message_start = conn->handshake.io.read_cursor;
```